apps/design/backend/src/worker/tasks.ts

```diff
@@ -8,13 +8,15 @@ import {
 
 import { BackgroundTask } from '../store';
 import { WorkerContext } from './context';
-import { generateElectionPackage } from './generate_election_package';
+import { generateElectionPackageAndBallots } from './generate_election_and_ballot_packages';
```

Comment:
nit: We can change the file name to match, too - `generate_election_package_and_ballots.ts`

Reply:
Definitely!